public/locales/en/common.json

```diff
@@ -149,7 +152,8 @@
       "LIKUTIM": "Assorted Excerpts",
       "SOURCE": "Kabbalah Library",
       "PICNIC": "Picnic",
-      "R_TWEET": "Tweets"
+      "R_TWEET": "Tweets",
+      "PUBLIC_EVENTS": "Public Events"
```

Comment:
Would be nice to use google translator and to translate it to different languages

Reply:
we do it for 4 main languages